sdk/python/packages/flet-cli/src/flet_cli/commands/create.py

```diff
@@ -35,21 +39,39 @@ def add_arguments(self, parser: argparse.ArgumentParser) -> None:
         parser.add_argument(
             "--template",
             dest="template",
-            choices=["minimal", "counter"],
```

Comment:
No more templates? What's the idea?

Reply:
It's been only one template in reality. We will add more when they appear.